paymentsheet/src/main/java/com/stripe/android/paymentsheet/viewmodels/PrimaryButtonUiStateMapper.kt

```diff
@@ -69,16 +70,16 @@ internal class PrimaryButtonUiStateMapper(
         return if (config.primaryButtonLabel != null) {
             config.primaryButtonLabel
         } else if (isProcessingPayment) {
-            val fallback = context.getString(R.string.stripe_paymentsheet_pay_button_label)
-            amount?.buildPayButtonLabel(context.resources) ?: fallback
+            val fallback = context.forLanguage().getString(R.string.stripe_paymentsheet_pay_button_label)
```

Comment:
I think we should update these to use `resolvableString` instead.

Reply:
Paired with Tyler yesterday on this but forgot to add an update on this PR. We built a solution with `ResolvableString` that solves this issue.

One thing that came out of it is that `ResolvableString` isn't parcelable which could affect future other migrations that may require it to be parcelable. Doesn't affect `PrimaryButton` but other areas of the app. To be done outside of this PR.

Reply:
Pushed @samer-stripe's commit and update tests 👍